alerts/src/androidLibMain/kotlin/AlertPresenter.kt

```diff
@@ -114,6 +146,54 @@ actual class AlertPresenter(
             }
         presentation.completion()
     }
+
+    /**
+     * Creates an [EditText] wrapped in a [LinearLayout] for dialogs of type [Alert.Style.TEXT_INPUT]
+     *
+     * @param context The Android context used to create view
+     * @param textInputAction The [Alert.TextInputAction] used for initializing the [EditText]
+     */
+    private fun createEditTextView(
+        context: Context,
+        textInputAction: Alert.TextInputAction
+    ): LinearLayout {
+        val linearLayout = LinearLayout(context)
+        val layoutParams = LinearLayout.LayoutParams(
+            ViewGroup.LayoutParams.MATCH_PARENT,
+            ViewGroup.LayoutParams.WRAP_CONTENT
+        )
+        val editText = EditText(context)
+        editText.layoutParams = layoutParams
+        linearLayout.addView(editText)
+        linearLayout.setPaddingRelative(16, 0, 16, 0)
```

Comment:
could you please use resources for the padding values?

Reply:
https://github.com/splendo/kaluga/pull/380/commits/e528b9a1cc0f9c1d38ea2f3dd15d5647c32c87ab